src/raids/Uldir/Zul.js

```diff
@@ -12,5 +12,7 @@ export default {
     // TODO: Add fight specific props
     // e.g. baseDowntime (seconds, percentage, based on (de)buff, etc)
     // e.g. ads
+    softMitigationChecks: {
```

Comment:
[Bloody Cleave](https://www.wowhead.com/spell=273316/bloody-cleave) (from the crusher) is a soft mit check.